engine/src/main/java/org/terasology/engine/rendering/assets/texture/TextureData.java

```diff
@@ -70,6 +72,16 @@ public TextureData(TextureData fromCopy) {
         }
     }
 
+    /**
+     * Look up the color of a given pixel in the base mipmap level.
+     */
+    public Colorc getPixel(int x, int y) {
```

Comment:
I'm not sure if we want this exposed, but the data was already public.

Reply:
In which sense was it public before? Could you derive the value manually just by using public methods?

Reply:
Yes: `Texture.getData()` is public, and `TextureData.getBuffers()` is public, so this function could be implemented anywhere using only public methods. But `Texture.getData()` has a comment saying `// TODO: Remove when no longer needed`, which is the main reason I'm cautious of this.